l10n_ch_fds_upload_sepa/README.md

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3

```

Comment:
Check the template, it has a bit evolved:

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst